developers/weaviate/model-providers/_includes/provider.generative.py

```diff
@@ -215,6 +215,54 @@
 # clean up
 client.collections.delete("DemoCollection")
 
+# START BasicGenerativeFriendliAI
+from weaviate.classes.config import Configure
+
+client.collections.create(
+    "DemoCollection",
+    generative_config=Configure.Generative.friendliai()
```

Comment:
shouldn't it be `friendli()`?

Reply:
I believe the module name itself should be the full name of our company.
Compared to API key or Baseurl, the module name is more like a standalone form than a descriptive form.

Reply:
We use `friendli` as a noun, short for `friendli suite`. 
Other integrations use `friendli` as framework, module, integration names

Reply:
Maybe I was confused since we used `friendliai`  at the latest integration LiteLLM. (https://docs.litellm.ai/docs/providers/friendliai)

+ Do you think the provider name should also be `friendli` instead of `friendliai`? (like in Langchain)
<img width="300" alt="Screenshot 2024-08-21 at 11 45 42 AM" src="https://github.com/user-attachments/assets/6b30f81d-2b6c-443a-a1b6-485c03562426">

Reply:
@snowman39 Integration 이름은 FriendliAI 로 합시다. 
package namespace, standalone 사용은 전부 FriendliAI, descriptive 사용은 friendli 로 해주세요.